hardware/qcom/audio/default/0001-hal-msm8974-Use-64bit-types-to-request-mute-volume-c.patch

```diff
@@ -0,0 +1,80 @@
+From 84a5856fe86101ecf809024e04e4ef9afb9e6e50 Mon Sep 17 00:00:00 2001
+From: Alfred Neumayer <dev.beidl@gmail.com>
+Date: Sun, 8 Aug 2021 18:32:18 +0200
+Subject: [PATCH] hal/msm8974: Use 64bit types to request mute & volume changes
+
+64bit kernels fail to apply mute & volume requests from userspace
+when the sizes don't match, since they can't be parsed.
+Turn them into 64bit types instead.
+
+Change-Id: I934f6a77147a56b43450d595e93e1f3cddea533f
+---
+ hal/Android.mk         |  2 +-
+ hal/msm8974/platform.c | 10 +++++-----
+ 2 files changed, 6 insertions(+), 6 deletions(-)
+
+diff --git a/hal/Android.mk b/hal/Android.mk
+index 88dbbe5..d1d804b 100644
+--- a/hal/Android.mk
++++ b/hal/Android.mk
+@@ -231,7 +231,7 @@ LOCAL_MODULE_TAGS := optional
+ 
+ LOCAL_MODULE_OWNER := qcom
+ 
+-LOCAL_PROPRIETARY_MODULE := true
++LOCAL_PROPRIETARY_MODULE := false
```

Comment:
Why this needs to be changed

Reply:
This causes it to store the module in system instead of vendor. Then a symlink can override which module to prefer to load. Allows running both with custom and pristine vendor images.